docs/io/ascii/write.rst

```diff
@@ -269,3 +269,95 @@ details.
   Note: Reader classes and Writer classes are synonymous, in other
   words Reader classes can also write, but for historical reasons they are
   often called Reader classes.
+
+
+
+.. _ecsv_format:
+
+ECSV format
+===========
+
+The `Enhanced Character-Separated Values (ECSV) format
+<https://github.com/astropy/astropy-APEs/blob/master/APE6.rst>`_ can be used to
+write astropy `~astropy.table.Table` or `~astropy.table.QTable` datasets to a
+text-only data file and then read the table back with loss of information.  The
```

Comment:
Looking at this again - did you mean that there might be slight loss of precision? If so, just update...